openrtb_ext/imp_between.go

```diff
@@ -1,5 +1,8 @@
 package openrtb_ext
 
 type ExtImpBetween struct {
-	Host string `json:"host"`
+	Host        string `json:"host"`
+	PublisherID string `json:"publisher_id"`
+	BidFloor    string `json:"bid_floor"`
```

Comment:
Are `bid_floor` and `bid_floor_cur` also params that you expect in the imp ext? Are they just optional parameters that you pass it to your bidder as is if present? If so, please add them to the `static/bidder-params/between.json` file as well as optional parameters